docker_setup_scripts/centos.sh

```diff
@@ -120,6 +128,22 @@ detect_centos_version() {
   readonly centos_major_version
 }
 
+add_repos() {
+  local repos=()
+  if [[ $centos_major_version -eq 7 ]]; then
+    repos+=( "${CENTOS7_ONLY_REPOS[@]}" )
+  elif [[ $centos_major_version -eq 8 ]]; then
+    repos+=( "${CENTOS8_ONLY_REPOS[@]}" )
+  else
+    echo "Unknown CentOS major version: $centos_major_version" >&2
+    exit 1
+  fi
+
+  for repo in "${repos[@]}"; do
+    (cd /etc/yum.repos.d/ && curl -O ${repo})
```

Comment:
Add double quotes around `${repo}`.

Also I would write this as:

```
(
  cd /etc/yum.repos.d
  curl -O "${repo}"
)
```
because we are already in the `set -e` mode which will exit in case `cd` fails.